app/controllers/user_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UserController < ApplicationController
+	skip_before_action :verify_authenticity_token, only: [:create]
```

Comment:
- Hàm này verify_authenticity_token có tác dụng gì
- Trình bày lại toàn bộ indent của file này

Reply:
Hàm verify_authenticity_token có tác dụng bỏ xác thực token cho action create ạ